src/com/generic/main/Shipment.java

```diff
@@ -0,0 +1,48 @@
+package com.generic.main;
+
+public class Shipment {
+	private String shipmentID;
```

Comment:
Shipment class looks like an immutable object and once instantiated, it can no longer be changed.
Maybe we can add an inner class Builder to return an immutable Shipment object